modules/virt-troubleshooting-cert-rotation-parameters.adoc

```diff
@@ -0,0 +1,33 @@
+
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-configuring-certificate-rotation.adoc
+
+[id="virt-troubleshooting-cert-rotation-parameters_{context}"]
+
```

Comment:
It renders fine, but I'd suggest removing the blank line between the ID and the title to match our guidelines: (https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#module-file-metadata)